scripts/auth_certificates

```diff
@@ -147,15 +143,15 @@ if [[ "$AUTH_TYPE" = "ca" ]]; then
   # Add more info /C=$country/ST=$state/L=$locality/O=$organization/OU=$organizational_unit/CN=$common_name/emailAddress=$email
   SUBJ="/C=${SSL_COUNTRY}/ST=${SSL_STATE}/O=${SSL_ORGANIZATION}/CN=${AUTH_NAME}/emailAddress=${SSL_EMAIL}"
 
-  openssl genrsa -out "${SSL_LOCATION}/${AUTH_NAME}.key" 4096 2> /dev/null
+  openssl genrsa -out "${SSL_LOCATION}/${AUTH_NAME}.key" "${KEY_SIZE}" 2> /dev/null
   openssl req -new -x509 -days "${SSL_EXPIRATION_DAYS}" -key "${SSL_LOCATION}/${AUTH_NAME}.key" -out "${SSL_LOCATION}/${AUTH_NAME}.cert" -subj '/C=US/CN=MY-CA' 2> /dev/null
```

Comment:
should we use "${SUBJ}"?

Reply:
For the CA certificate, the subject is restricted to `/C=US/CN=MY-CA`; to prevent any regression, I am limiting it solely to the required changes.